elastic/kibana
# [Response Ops] [Alerting] Run rule after enabling

Resolves https://github.com/elastic/kibana/issues/140632

## Summary

Splits the task manager `bulkEnableDisable` function into `bulkEnable` and `bulkDisable` and updates `bulkEnable` to allow a `runSoon` parameter to determine whether to reset the tasks's `runAt` and `scheduledAt` so that it runs sooner. 

## To Verify

Create a rule that runs on a longer interval like 1 day. Let it run once and then disable and re-enable it. Verify it runs again right after enabling.


### Checklist
